cico_setup.sh

```diff
@@ -24,19 +24,16 @@ build_image() {
 tag_push() {
     local target=$1
     local source=$2
+    echo "docker tag ${source} ${target}"
     docker tag ${source} ${target}
     docker push ${target}
 }
 
 push_image() {
-    local image_name
-    local image_repository
-    local tag
-    local push_registry
-    image_name=$(make get-image-name)
-    image_repository=$(make get-image-repository)
-    tag=$(git rev-parse --short=7 HEAD)
-    push_registry="push.registry.devshift.net"
+    local image_name=$(make get-image-name)
```

Comment:
Note there was a reason why the variable declaration and assignment was not one-liner before. We execute the script with `-e`, i.e. we want it to fail early if there are any problems. Declaration of a local variable will always succeed here, so it can hide `make` failures.